sql/pre_install/tables.sql

```diff
@@ -180,6 +180,7 @@ CREATE TABLE IF NOT EXISTS _timescaledb_internal.bgw_job_stat (
     last_start              TIMESTAMPTZ NOT NULL DEFAULT NOW(),
     last_finish             TIMESTAMPTZ NOT NULL,
     next_start              TIMESTAMPTZ NOT NULL,
+    last_successful_finish  TIMESTAMPTZ NOT NULL,
```

Comment:
OK this is defined NOT NULL but the output below shows this without value

Reply:
The output below is from a view with a left join on this table